app/controllers/api/v1/users_controller.rb

```diff
@@ -21,6 +21,15 @@ def create
         end
       end
 
+      def update
+        user = User.find(params[:id])
+        if user.update(user_params)
+          render_json status: :ok
```

Comment:
To further simplify this, render_json by default returns render json: { data: [], errors: [] }, status: :ok, include: nil

Reply:
I would still keep the `status: :ok` so that its clear that this is a success

Reply:
Also - shouldn't we return the updated user?

Reply:
Thats fine for now - i'll look into it later